### PR TITLE
EXPAND YOUR GREMLIN

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 fluffe9911 for porting steamp0rt for actually coding it
add: gremlins are nonsentient mobs that can now randomly appear around the station and screw around with electronics!
/🆑

see https://github.com/HippieStation/HippieStation/pull/3098 for more details

### CODE REVIEW

**Comment by ghost** [comment]
is your idea list literally just searching hippie's repo for `author:steamp0rt`, @fluffe9911

**Comment by ThatLing** [comment]
@steamp0rt stop ban evading

@fluffe9911 lower event chance

**Comment by ghost** [comment]
@ThatLing let me appeal then. 


what if i port slings

**Comment by fluffe9911** [comment]
@steamp0rt Nah I just look threw random codebases for things I like and it just so happens hippie not only has a ton of N E A T ideas its also extremely easy to port since both yogs and hippie stay up to date with tg also mcdonald already ported slings ( gangs isnt ported yet though)

@ThatLing done

**Comment by ghost** [comment]
@fluffe9911 oh nice he used my shadowling code... without credit

**Comment by fluffe9911** [comment]
Yeah guess he didn't think to *shrug*
edit or he didnt know you were the original author caused he used someone elses who tried to port it in the first place and didnt realize he took that from you since the first guy never credited you one of the two I guess *shrug*

File: yogstation/code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -73,7 +73,15 @@
 	item = /obj/item/storage/box/syndie_kit/cluwnification
 	cost = 25
 	restricted_roles = list("Clown", "Cook")
+	surplus = 5
```
Comment (ThatLing) [inline yogstation/code/modules/uplink/uplink_item.dm:76]: no

File: yogstation/code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -73,7 +73,15 @@
...
 
 /datum/uplink_item/device_tools/hypnotic_flash
 	restricted_roles = list("Psychiatrist")
 	category = "Role-Restricted"
+
+/datum/uplink_item/dangerous/gremlin
+	name = "Gremlin delivery grenade"
+	desc = "This grenade is filled with several gremlins. Fun for RnD and engineering!"
+	cost = 2
```
Comment (ThatLing) [inline yogstation/code/modules/uplink/uplink_item.dm:85]: up this

File: yogstation/code/modules/mob/living/simple_animal/gremlin/event.dm [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/datum/round_event_control/gremlin
+	name = "Spawn Gremlins"
+	typepath = /datum/round_event/gremlin
+	weight = 10
```
Comment (ThatLing) [inline yogstation/code/modules/mob/living/simple_animal/gremlin/event.dm:4]: Lower this

File: yogstation/code/modules/mob/living/simple_animal/gremlin/event.dm [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	max_occurrences = 2
```
Comment (ThatLing) [inline yogstation/code/modules/mob/living/simple_animal/gremlin/event.dm:5]: 1

**Review by ThatLing — CHANGES_REQUESTED** [review]

**Comment by ThatLing** [comment]
@steamp0rt why can't you appeal?

**Comment by fluffe9911** [comment]
@ThatLing you are aware these are simple mobs and not naturally player controlled right?

**Comment by ghost** [comment]
@ThatLing i dunno i just assumed i can't. i guess i can

**Pull request closed without merging**